.husky/pre-commit

```diff
@@ -1,6 +1,18 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-yarn format
-yarn lint
-yarn type-check
+## Run lint-staged
```

Comment:
Just out of curiosity: why is `##` used here instead of `#`?

Reply:
Nothing special, it's a personal habit learned from MATLAB. I use it to signal a section in shell scripts